<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicSettingsWindow</class>
 <widget class="QWidget" name="BasicSettingsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>189</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Basic settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="project-721.qrc">
    <normaloff>:/images/settings-128.png</normaloff>:/images/settings-128.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Alarm temperature</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Above</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="tempAbove">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="tempBelow">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Below</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>°C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>°C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Alarm on HW errors</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="hwErrors">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QCheckBox" name="clear_firmware_update">
     <property name="text">
      <string>Clear up device settings when firmware update</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QCheckBox" name="log_enabled">
     <property name="text">
      <string>Enable logging</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="project-721.qrc"/>
 </resources>
 <connections/>
</ui>
